src/iconsManifest/supportedExtensions.ts

```diff
@@ -3342,7 +3342,7 @@ export const extensions: IFileCollection = {
         'rollup.config.dev',
         'rollup.config.prod',
       ],
-      extensionsGlob: ['js', 'mjs', 'coffee', 'ts'],
+      extensionsGlob: ['js', 'cjs', 'mjs', 'coffee', 'ts'],
```

Comment:
Sorry but I'm not sure why we should support `cjs` as an extension. According to the PR you refer to I can only see that the output uses the `cjs` format but it doesn't mention the extension. 

Can you elaborate a little bit more?

Reply:
Yes sure, please take a look at this link https://rollupjs.org/guide/en/#configuration-files

It says you can create a file config with `.js`, `.cjs` and `.mjs` extensions.

Referring to `.cjs` extension `will prevent Rollup from trying to transpile the file` and read it directly as a Node.js script.

Reply:
Thanks!